cachi2/core/package_managers/bundler/main.py

```diff
@@ -26,3 +31,30 @@ def _resolve_bundler_package(package_dir: RootedPath, output_dir: RootedPath) ->
     """Process a request for a single bundler package."""
     parse_lockfile(package_dir)
     return []
+
+
+def _prepare_for_hermetic_build(output_dir: RootedPath) -> None:
+    """Prepare a package for hermetic build by injecting necessary config."""
+    bundle_config = output_dir.join_within_root(".bundle/config").path
+    hermetic_config = dedent(
+        f"""
+        BUNDLE_CACHE_PATH={output_dir}/deps/rubygems
```

Comment:
`output_dir` is the absolute path where cachi2 stores the artifacts, but that path won't exist in the build environment, that path is determined by the CLI option `--for-output-dir` which injects the path to these env variables which we know will appear in the container. The way you apply that in code is by using `${output_dir}` which is a pattern that's going to be replaced either by the `generate-env` or `inject-files` command, we use Python's `Template` for those substitutions.

Reply:
I did not realize that, thank you!